_posts/2016/01/2016-01-14-steering-raniere-silva.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+author: "Raniere Silva"
+title: "2016 Election: Raniere Silva"
+date: 2016-01-14
+time: "15:00:00"
+category: ["Software Carpentry Foundation"]
+---
+My name is Raniere Gaia Costa da Silva
+and I'm standing for re-election to Software Carpentry Steering Committee
+because working on the Steering Committee in 2016 was fun
+and a great way to meet some of the amazing members of Software Carpentry community.
+
+## Background
+
+I have a B.Sc. in Apply Mathematics from the University of Campinas, Brazil,
+and in the last year I worked as a freelancer software developer
+using most of the time Python and Javascript.
+This year I will move to the UK for work on the University of Manchester
+and collaborate with The Software Sustainability Institute.
+
+## Previous Involvement
+
+I discovered Software Carpentry in 2013 (I wish that I did it early)
+and contacted Greg asking to be in the next round of instructor training,
+what happened in 2013's Fall, because I was trying to do Software Carpentry like workshops in Brazil.
+
+In 2014 I sent some pull requests to the lessons
+(mostly to Shell, Git and Python),
+annoyed Greg because we use Python 2 and not Python 3,
+helped over-engineering our <del>crazy</del> lesson template,
+and delivery the first half dozen of Software Carpentry in Brazil.
+
+Last year,
+because of Steering Committee activities
+I significant drop the number of pull request sent to the lessons
+(something that I missed very much).
+On the Steering Committee
+my biggest contribution was leading the Mentoring Subcommittee
+that host the post workshop debriefing sessions
+and the help & instructor retreat that happened last Fall.
+If you want to read more about what I was involved in 2015
+check [this blog post](http://blog.rgaiacs.com/2015/12/11/swc.html).
+
+## Plans for 2016
+
+In 2016,
+independent of the result of the Steering Committee election result,
+I want to focus on the assessment of achievements that our students had after
+attend our workshops because this is a important piece to Software Carpentry
+sustainability and expansion.
+
+I will not leave the Mentoring Subcommittee
+because it is one of the ways to get data for assessments
+but I hope to pass the leadership of it to one of our current members.
+
+If I was elected I want be secretary of 2016 Steering Committee
+because although we did a good job with the minutes of the meetings
+I believe that we could do it a lot better.
+
+A few extra things that I want to do but don't plan to spend many hours are:
+
+-   run our first workshop in Portugal (or any other country that speaks Portuguese),
+-   collaborate to workshops in Latin America continue to be offered, and
+-   translate the lessons to Portuguese (or any other language).
+
+## More
+
+Sometimes I write in [my blog](http://blog.rgaiacs.com/) that has posts about
```

Comment:
Maybe “Sometimes post to [my blog](http://blog.rgaiacs.com/) about books that I read, cities I visit, …”?